skimage/_build.py

```diff
@@ -69,3 +74,21 @@ def _changed(filename):
             cf.write(md5_new.encode('utf-8'))
 
     return md5_cached != md5_new.encode('utf-8')
+
+
+def process_tempita_pyx(fromfile):
+    try:
+        try:
+            from Cython import Tempita as tempita
+        except ImportError:
+            import tempita
+    except ImportError:
+        raise Exception('Building requires Tempita: '
+                        'pip install --user Tempita')
+    from_filename = tempita.Template.from_filename
```

Comment:
Not really necessary to do this re-assignment, since we don't use it more than once further down